src/plugins/vis_type_vega/public/components/vega_help_menu.tsx

```diff
@@ -34,13 +34,18 @@ import React, { useCallback, useState } from 'react';
 import { EuiButtonIcon, EuiContextMenuPanel, EuiContextMenuItem, EuiPopover } from '@elastic/eui';
 import { FormattedMessage } from '@osd/i18n/react';
 import { i18n } from '@osd/i18n';
+import { useOpenSearchDashboards } from '../../../opensearch_dashboards_react/public';
 
 function VegaHelpMenu() {
   const [isPopoverOpen, setIsPopoverOpen] = useState(false);
   const onButtonClick = useCallback(() => setIsPopoverOpen(!isPopoverOpen), [isPopoverOpen]);
 
   const closePopover = useCallback(() => setIsPopoverOpen(false), []);
 
+  const opensearchDashboards = useOpenSearchDashboards();
+  const opensearchDashboadsDoc = opensearchDashboards.services.docLinks!.links.opensearchDashboards
+    .introduction;
```

Comment:
It might be better to add a `vega` property to the `noDocumentation` object?

Reply:
Yes, we can.